### PR TITLE
Update SugarFolders.php Not returning User Accessible inbound accounts (group)

### PR DESCRIPTION
**Pull request by pstevens71**
Just troubleshooting group folders. and found this.

2 small bugs: 

## Description

1) $item->isgroup is not correct.  It is not a class but and array and should be retrieved like $item['is_group'] also the name isgroup is incorrect, it should be is_group like in the array.

2) if($isGroup===1) is too restrictive.  Should be if ($isGroup == 1) which will return true.  (this was an issue in 7.12 too!)

Since Group folders are not working anyway (I'm working on that too! the problem is the JSON returns group folders in a separate array for the folders selections in user profile list, I haven't tested potential impacts of changing this)

## Motivation and Context

Trying to get group folders working and usable for users.  Currently cannot see or view group folders.

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: public/legacy/include/SugarFolders/SugarFolders.php
Comment (pstevens71) [inline public/legacy/include/SugarFolders/SugarFolders.php:1]: This change is no longer necessary, I have incorporated this in a revised retrieveFoldersForProcessing function that includes this fix along with functionality to check if current user is admin and also to filter out group folders that the current user does not have access to.  Currently, all group folders show.

Comment (clemente-raposo) [reply public/legacy/include/SugarFolders/SugarFolders.php:1]: Hi @pstevens71, thank you for the PR and for the updated.

I didn't follow 100%.

Does this mean we can close this PR?

Comment (pstevens71) [reply public/legacy/include/SugarFolders/SugarFolders.php:1]: Yes you can close this one.  I'm going to enter another one today that includes this change plus a few other things in this particular function to support getting group folders to work properly.  I have a whole bunch of changes that together solve the whole problem.  Apologies I'm kind of new to Github and don't know how to delete or modify this PR.  After I entered it I realized a bunch of other modifications where necessary in addition to this.

**Comment by clemente-raposo** [comment]
@pstevens71 thank you.

closing according to https://github.com/salesagility/SuiteCRM-Core/pull/276#discussion_r1221522066

**Pull request closed without merging**